### ISSUE TITLE
Http get_message fails when a message has reactions

### ISSUE THREAD

**Issue by Woonters** [serenity-rs/serenity#3022]
If using the http method `get_message` on a message with reacts the error "missing field `burst_colours` at line 1 column 792" is returned, messages without react images return Ok(...) this can be seen with my minimal working example [here](https://github.com/Woonters/serenity_http_get_message_bug).

I'm happy to look at writing a fix for it but wanted to raise this before starting any work

**Comment by GnomedDev** [comment]
Duplicate of #2892
